src/includes/classes/MenuPageOptions.php

```diff
@@ -251,9 +251,9 @@ public function __construct()
         /* --------------------------------------------------------------------------------------------------------------------------------------------------------------------------- */
 
         if (IS_PRO || $this->plugin->isProPreview()) {
-            echo '<div class="plugin-menu-page-panel'.(!IS_PRO ? ' pro-preview' : '').'">'."\n";
```

Comment:
@jaswsinc Actually, the way this worked changed awhile back, as part of the improvements to the Pro Preview. What @renzms did here is correct.